### ISSUE TITLE
Update source code

### ISSUE THREAD

**Issue by hsklia** [peter-iakovlev/Telegram#195]
@Peter-iakovlev Why you are not updating the source code?
How you can say Telegram is “Open Source”?

**Comment by d-Rickyy-b** [comment]
Source code was just updated 30 minutes ago ;)

**Comment by jvschiavo** [comment]
@d-Rickyy-b is it from version 4.6? I'm not sure how to see that.

**Comment by Shiaulis** [comment]
@peter-iakovlev is there any issues with updating submodules in repo? I have an error on 'git pull --recurse-submodules' command:
Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

**Comment by d-Rickyy-b** [comment]
@jvschiavo Yes, it's the code of v4.6 - https://github.com/peter-iakovlev/Telegram/blob/90cce8fcdcff9bdbf5e757386618fecc42c95137/LegacyDatabase/Info.plist#L18
